megatron/model/transformer.py

```diff
@@ -118,19 +114,16 @@ def squared_relu(x):
             self.bias_gelu_fusion = args.bias_gelu_fusion
             self.activation_func = F.gelu
 
-        # Project back to h.
-        self.dense_4h_to_h = tensor_parallel.RowParallelLinear(
-            config.ffn_hidden_size,
-            config.hidden_size,
-            config=config,
-            init_method=config.output_layer_init_method,
-            bias=self.add_bias,
-            input_is_parallel=True
+        self.dense_4h_to_h = Tensor_Parallel_Linear(
+            in_features=config.ffn_hidden_size,
+            out_features=config.hidden_size,
+            skip_bias_add=True 
         )
 
     def forward(self, hidden_states):
 
         # [s, b, 4hp]
+        hidden_states = drop(hidden_states)
```

Comment:
Divides the hidden states in the right configuration for AxoNN's tensor parallelism. The input to drop should be replicated on all tensor parallel ranks.